tests/config/test_storages.py

```diff
@@ -63,7 +63,11 @@ def _read_and_write_metadata(self, ursula, node_storage):
         all_stored_nodes = node_storage.all(federated_only=True)
         all_known_nodes.add(ursula)
         assert len(all_known_nodes) == len(all_stored_nodes)
-        assert all_stored_nodes == all_known_nodes
+
+        known_checksums = sorted([n.checksum_address for n in all_known_nodes])
```

Comment:
I think, `[` and `]` are not necessary here

Reply:
Indeed; good catch.  I will include this change in #1881.